.github/workflows/ci.yml

```diff
@@ -107,7 +107,7 @@ jobs:
       # dependent jobs.
       build-dir: ${{ env.BUILD_DIR }}
       build-dir-archive: ${{ steps.archive.outputs.build-dir-archive }}
-      gcs-artifact: ${{ steps.upload.outputs.gcs-artifact }}
+      build-dir-gcs-artifact: ${{ steps.upload.outputs.build-dir-gcs-artifact }}
```

Comment:
I was keeping this terse with the rationale that which gcs-artifact it is is already implied by the job name and this can only be referenced relative to the job name. I guess the same could be said for "build-dir-archive" here, which could just be "archive". Not sure that "dir" really stands on its own though.

Reply:
I was thinking about generating `build-dir-archive` and `install-dir-archive`, at which point `gcs-artifact` would be ambiguous. I ended up just using the single archive, but then figured the naming clarification / consistency would be useful regardless.

Reply:
Yeah I don't have super strong opinions. I think there's some value to brevity when it's already namespaced, but :shrug: